server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -124,11 +127,11 @@ object NettyConfig {
     serverHeader = Some(s"tapir/${buildinfo.BuildInfo.version}")
   )
 
-  def defaultInitPipeline(cfg: NettyConfig)(pipeline: ChannelPipeline, handler: ChannelHandler): Unit = {
+  def defaultInitPipeline(cfg: NettyConfig)(pipeline: ChannelPipeline, handlers: List[ChannelHandler]): Unit = {
     cfg.sslContext.foreach(s => pipeline.addLast(s.newHandler(pipeline.channel().alloc())))
-    pipeline.addLast(new HttpServerCodec())
-    pipeline.addLast(new HttpStreamsServerHandler())
-    pipeline.addLast(handler)
+    pipeline.addLast(ServerCodecHandlerName, new HttpServerCodec())
+    pipeline.addLast(StreamsHandlerName, new HttpStreamsServerHandler())
+    handlers.foreach(pipeline.addLast(_))
```

Comment:
it doesn't matter that the handlers are now not named?

Reply:
Names are useful if you want to explicitly add other handlers before or after specific handlers. There's no such need in case of these two.